### DISCUSSION TITLE
failed to open the xaml

### DISCUSSION THREAD

**Discussion by ZIHAOJIA** [MichaelKelly9/Tafe24S1T104_Ext#4] [General]
when I download the file and open oit through visual studio 2022, I can only see the xaml.sc code but can not see the xaml，it shows Some assembly references are missing.

**Comment by MichaelKelly9** [comment]
Hi @ZIHAOJIA 
It looks like you are working on the main branch not the dev branch. After cloning you need to checkout the origin/development branch to see the updated code. I think you also need to ensure create branch is ticked during checkout and you will then have the deveopment branch as a local copy to work on.

Let me know if that fixes the issue

**Reply by ZIHAOJIA** [reply]
yes , i try to download using zip ,but failed . i also try to use git clone but need public key ,i will try to use git sync once more

**Reply by MichaelKelly9** [reply]
Sorry I forgot to add your public key. Can you please post the public key here? it will be easier for me to copy and paste it into the settings

**Reply by ZIHAOJIA** [reply]
thanks for you advice, i successful entered the dev branch and find the main menu. my pub key is SHA256:UpEEbXwzjQMZVQi6NDVPMvM0KJsW2oyIkJsx5Hi3ubc

**Reply by ZIHAOJIA** [reply]
i created a new file and git clone and then succeeded. Last time I failed due to clone it in a old file.

**Reply by MichaelKelly9** [reply]
No worries, I think you only need the public key for a push anyway. 

You will have to send me the full version key though, similar to the one below. 

ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCF0+N5WFbsGel2Xl23ww8toRQPvBz1Q1L6yfRVOCrfKkO7iTT8cO28vuqQPVhk7VRxFNz1JLWWeQcg55u6PVfBcXprkhqqkg/SD27lx+eg3VJmOmIug60oCoVQhnSGMK1Ydm+xwV1+6LtHaj8tGSyF5JiqemXlKmTCZakW690TCP1R9nUvNaHkF+8xahHn02NJLoysvsHi4sY8G94AxPvPqM5+PDxeq0LyoUjdq7sFBGMOKV/Os6W5DQqFUQgR8A/H6kV+ctTZlAvMgK95hOD2+CA5Q2RAufr9o1Rggl/zY9zujntRe0F31/XCWrTOYsiMSo69GpQARxAlmjEHJ/wb rsa-key-20240221

**Reply by ZIHAOJIA** [reply]
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCmUIBV2nrM9qgeP2M/DzWmiH2ImHxsbGN0IVmthKgQQ539uJ4BreeJr/HNFX9CPWRV1mWi/XDH4BazJ7lklLXJiXvuh2tnjeDyiFXfH1kjczMlPzwYKoTT7fS/HMUmsJaekJa8AyrI5iEsB3XH29Hluw9+8RXiyEOdK7wLd5QChZf9zzyN6t078MUP/+SA0uWSaEcS65pTkmkgODRFA7g8wHq/ysDp/vw+wd990uJr3TWOqE/XIcxz996gGt3AMswfWF0oo4MB25YAkoQ9YuSas74Bh4bzf8TKLmncjNjHGhY7onkfKLwFoK/inBpGWTzA6KIqSBqocYvN98Qv1ulp rsa-key-20240301

**Reply by MichaelKelly9** [reply]
Thanks Zihao that's what i needed.

Although I will have to check with KT how to correctly setup the Keys as it is displaying "key already in use" which makes sense if you have already added it to your account. 

I'm not sure if you would need to delete it from your account so I can add it to mine or not. I am a bit confused around this Key pair thing.

Anyway shouldn't impact us until you need to push/pull the repo

**Reply by ZIHAOJIA** [reply]
when I TRY PUSH now ,it said can not load the key. i just use my key to upload it .i do not know whether it succeed or not.

**Reply by ZIHAOJIA** [reply]
I think when we are in this group ,we can use our own key to upload and download the file without using other people`s key.

**Reply by MichaelKelly9** [reply]
It all looks good, GitHub now has the feature2 branch.

**Reply by MichaelKelly9** [reply]
did you put in the private key file in the tortoisegit settings?
![image](https://github.com/MichaelKelly9/Tafe24S1T104_Ext/assets/160606357/4566b9c0-0ac2-4c22-9c5f-ed2b6aef16c1)
